src/legacy/java/net/neoforged/moddevgradle/legacyforge/dsl/ObfuscationExtension.java

```diff
@@ -175,19 +173,12 @@ public Configuration createRemappingConfiguration(Configuration parent) {
             // Additionally, we force dependencies to be non-transitive since we cannot apply the attribute hack to transitive dependencies.
             spec.withDependencies(dependencies -> dependencies.forEach(dep -> {
```

Comment:
Hmm. I... _really_ don't like this because it relies on the `modXyz` configuration being non-lazy -- which it hopefully will not be! Instead, this whole thing should probably be a `parent.getDependencyConstraints().addLater(...)` block, _outside_ of (after) the `create` block for `remappingConfig`, probably `.map`-ing `remappingConfig.getDependencies()`, so that this still works if remappingConfig is lazy.